runpod/serverless/modules/rp_scale.py

```diff
@@ -99,17 +99,26 @@ async def get_jobs(self, session):
             if not self.is_alive():
                 break
 
-            tasks = [
-                asyncio.create_task(
-                    get_job(session, retry=False)
-                ) for _ in range(self.num_concurrent_get_job_requests)]
-
-            for job_future in asyncio.as_completed(tasks):
-                job = await job_future
-                self.job_history.append(1 if job else 0)
-
-                if job:
-                    yield job
+            sequential = True
```

Comment:
If you are setting this to True, when can it ever be false? This would make the if condition useless right?